Makefile

```diff
@@ -54,13 +48,13 @@ install-local: ## Install BentoML in editable mode
 	@pip install --editable .
 install-dev-deps: ## Install all dev dependencies
 	@echo Installing dev dependencies...
-	@pip install -r requirements/dev-requirements.txt
+	@pip install "bentoml[development]"
```

Comment:
I'd prefer using requirements file, the dev/test/docs dependencies are not user facing, we should not put them in `[extra]`.

Reply:
sg